spec/controllers/admin/posts_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+
+require 'rails_helper'
+
+describe Admin::PostsController do
+
+  before (:each) do
+    @user = FactoryGirl.create :user, :admin
+    sign_in @user
+  end
+
+  describe 'GET' do
+
+    let(:post) { FactoryGirl.create :post }
+
+    describe 'GET index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.